### DISCUSSION TITLE
Horse como Serviço do Windows

### DISCUSSION THREAD

**Discussion by alissonrodrigo** [HashLoad/horse#400] [General]
Boa tarde procurei aqui e na internet a documentação ou exemplos de como rodar o horse como um serviço padrão do windows, mas não encontrei nada... 
Tenho uma aplicação horse rodando certinho via aplicação mas quando converto para um serviço Windows simplesmente não roda. E possível fazer isso usando Delphi? Grato

**Comment by viniciussanchez** [comment]
Boa tarde... sim é possível. Na pasta Samples tem exemplo

**Comment by alissonrodrigo** [comment]
Sim e verdade não tinha reparado me desculpe. Mas tenho um 'problema' com os testes que fiz. Se eu tenho outros TDataModules e adiciono ao projeto registrando as rodas ele não chama da
"Access violation at address 00AE9D98 in module 'Servico.exe'. Read of address FFFFFFFC"
mesmo eles sendo criados certinho... As unicas rotas que funciona são as que eu adicionei no "TServicoRenova = class(TService)" no evento onCreate do serviço
